bower.json

```diff
@@ -22,12 +22,12 @@
     "wct.conf.js"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0-rc.1"
+    "polymer": "Polymer/polymer#^v2.0.0"
```

Comment:
No reason, I was just haphazardly trying to fix dependencies to be somewhat close to correct.